Xamarin.Forms.Platform.iOS/Controls/Snackbar/SnackbarViews/BaseSnackbarView.cs

```diff
@@ -0,0 +1,130 @@
+﻿using UIKit;
+using Xamarin.Forms.Platform.iOS.Controls.Snackbar.Extensions;
+
+namespace Xamarin.Forms.Platform.iOS.Controls.Snackbar.SnackbarViews
+{
+	public class BaseSnackbarView : UIView
```

Comment:
The code style is wrong, we use tabs, you can also remove the summary.

I think we should do this internal.